EntityFramework/HomeWork1/Models/Courier.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace FoodOrderingSystem.Models
+{
+    public class Courier
```

Comment:
seems like better to create User model 
and derive Courier and Customer from User